test/functional/tools/MountebankService.ts

```diff
@@ -91,7 +91,6 @@ export class MountebankService {
               {
                 proxy: {
                   to: `http://localhost:${this.options.imposterPort}`,
-                  mode: 'proxyTransparent',
```

Comment:
just curious, what does this do?

Reply:
The `removeImposters()` function is used to teardown the mock Gate server after a test is finished.  This property isn't one of the required parameters when doing this and was probably left over from an old copy/paste so I just removed it.